src/java.base/share/native/libjava/jni_util.c

```diff
@@ -462,7 +462,7 @@ static jstring
 newString646_US(JNIEnv *env, const char *str)
 {
     int len = (int)strlen(str);
-    jchar buf[512];
+    jchar buf[512] = {0};
```

Comment:
Seems harmless, but a spurious warning - which is to be expected I guess since the flag says "maybe".